paper/paper.md

```diff
@@ -18,119 +18,121 @@ authors:
+SPH was originally developed by [@Monaghan:1977] to simulate astrophysical applications and is now widely used to simulate computational fluid dynamics, structural mechanics and even heat conduction problems.
 
-Here, we give a brief overview of the software TrixiParticles.jl, starting with a description of the scientific background before going on to describe the functionality and usefulness in more detail.
-Finally, some of the results obtained and some of the features that have been implemented so far are briefly presented.
+Here, we give a brief overview of the software package TrixiParticles.jl, starting with a description of the scientific background before going on to describe the functionality and benefit in more detail.
+Finally, a few exemplary results and implemented features are briefly presented.
 
 # Statement of need
 
-Numerical simulations, such as computational fluid dynamics, structural mechanics, thermodynamics, or magnetohydrodynamics, are often conducted using mesh-based methods supported by mature software tools like [ANSYS Fluent](https://www.ansys.com/de-de/products/fluids/ansys-fluent), [COMSOL Multiphysics](https://www.comsol.com/), and [OpenFOAM](https://www.openfoam.com/). However, these methods face limitations for complex geometries or when integrating different physics disciplines into a single multiphysics system.
+Numerical simulations, such as computational fluid dynamics, structural mechanics, thermodynamics, or magnetohydrodynamics pose several challenges, especially when simulating real-world problems.
+These simulations involve challenges such as dealing with complex geometries, free surfaces,
+deformable boundaries and moving material interfaces as well as the coupling of different physics disciplines.
 
-The mesh-free formalism of the methods allows for easy preprocessing, making them particularly
-suitable for simulating complex geometries. This also facilitates the coupling of different single
-physics into a multiphysics system.
+The Lagrangian formalism in particle-based methods allows particles to move along a velocity field without any connection to neighboring particles, thus eliminating the need for a mesh to discretize the simulation domain.
+This mesh-free approach not only simplifies the preprocessing, making it particularly suitable for simulating complex geometries, but it also facilitates simulations of large deformations and movements.
+In addition, particle-based methods are inherently suited to simulating free surfaces, material interfaces and challenges involving moving boundaries.
+Facilitating the transition from continuum to fragmentation becomes straightforward by representing each material with its own set of particles, effectively addressing material interface issues.
+This also simplifies the coupling of different singlephysics into a multiphysics system.
```

Comment:
singlephysics? Maybe single-physics?
```suggestion
This also simplifies the coupling of different single-physics into a multi-physics system.
```

Reply:
I struggled too. But "multiphysics" is also written without a hyphen (e.g https://en.wikipedia.org/wiki/Multiphysics_simulation)

Reply:
I don't think "singlephysics" is ever used. I'd write something like
> [...] coupling of different physical systems into a multi-physics setup.